win32ss/gdi/ntgdi/freetype.c

```diff
@@ -4612,7 +4612,6 @@ ftGdiGetTextMetricsW(
                 FillTM(&ptmwi->TextMetric, FontGDI, pOS2, pHori, !Error ? &Win : 0);
 
                 /* FIXME: Fill Diff member */
-                RtlZeroMemory(&ptmwi->Diff, sizeof(ptmwi->Diff));
```

Comment:
This is called from places other than `NtGdiGetTextMetricsW`, so it doesn't seem safe to remove this.
Or actually maybe just zero the entire ptmwi in this function, then all callers will benefit.

Reply:
Thanks. I have followed your instructions.